### ISSUE TITLE
Commend Bot Patched Again

### ISSUE THREAD

**Issue by Raxana7** [BeepIsla/csgo-commend-bot#597]
Everyone searching for a fix...

**Comment by verdammnis** [comment]
no one is looking for it, no one cares.

and as stated by valve it was "forever fixed"

**Comment by trustyuy** [comment]
ı found fix

**Comment by trustyuy** [comment]
no one cares? you very brainless

**Comment by verdammnis** [comment]
> no one cares? you very brainless


people who make money from stupid people like you, they care. because they make money from stupid idiots.


and all the other people who are not interested in it, because they are not doing it for money = they do not care.

**Comment by emreires** [comment]
> no one is looking for it, no one cares.
> 
> and as stated by valve it was "forever fixed"

![image](https://user-images.githubusercontent.com/25120245/90621134-fd4e8f80-e21b-11ea-960a-7d0d5f8da17f.png)
![image](https://user-images.githubusercontent.com/25120245/90621281-2d962e00-e21c-11ea-8bcc-77be195b4532.png)

Obv. everyone care this :)

There is Github. Even if 1 person cares it, he/she can create a new issue and talk with people.

**Comment by verdammnis** [comment]
> > no one is looking for it, no one cares.
> 
> > 
> 
> > and as stated by valve it was "forever fixed"
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/25120245/90621134-fd4e8f80-e21b-11ea-960a-7d0d5f8da17f.png)
> 
> ![image](https://user-images.githubusercontent.com/25120245/90621281-2d962e00-e21c-11ea-8bcc-77be195b4532.png)
> 
> 
> 
> Obv. everyone care this :)
> 
> 
> 
> There is Github. Even if 1 person cares it, he/she can create a new issue and talk with people.


issue was closed because the person shared the advice. 

I agree, beepisla is interested in helping, but he doesn't care about fixing it. he doesn't care, because he is engaged in other projects :) 


in any case, this is not considered a solution to the problem. I just gave you my opinion.

**Comment by emreires** [comment]
Thats why people is creating new issues. The problem isnt BEEPISLA. Someone else can fix and release it :)

Please learn how to use Github.

@Raxana7 I'm dumping why the commends can't send, I will share the log docs and dumps in soon.

**Comment by Nasri69** [comment]
@BeepIsla 
thats the only change valve made to csgo
optional .CMsgGCCStrike15_v2_MatchmakingGC2ServerReserve reservation = 2;

**Comment by RadicalRedemption** [comment]
First 20 fail, second 20 go thru successfully. It needs another preloading i guess.

**Comment by lightbgs** [comment]
![image](https://user-images.githubusercontent.com/31590513/90633460-750cc200-e1fc-11ea-9d1e-3f5e076d4f74.png)

**Comment by trustyuy** [comment]
jaeger is selfish he dont tell fix

**Comment by trustyuy** [comment]
ı found fix but only 3 succesfull commends 17 failed

**Comment by trustyuy** [comment]
ı got 12 commends now

**Comment by Amerrrrr** [comment]
<img width="759" alt="image" src="https://user-images.githubusercontent.com/58820550/90637370-e6974100-e1f9-11ea-8a1c-617bcc71c8b6.png">

**Comment by ninjaibiss** [comment]
@Amerrrrr  help us plz :(

**Comment by lightbgs** [comment]
> ı found fix but only 3 succesfull commends 17 failed

what you did?

**Comment by lightbgs** [comment]
> @Amerrrrr help us plz :(

it was good if @Amerrrrr help, but i don't think he will :(

**Comment by ninjaibiss** [comment]
@lightbgs do you have a discord commendbot?

**Comment by ninjaibiss** [comment]
@lightbgs  i have a discordcommendbot but still on beta maybe u can help me improve it?

**Comment by trustyuy** [comment]
![image](https://user-images.githubusercontent.com/65316785/90642569-3d713a80-e23b-11ea-890c-9b54b206fa5d.png)
it works ı will made a tutorial

**Comment by ninjaibiss** [comment]
@trustyuy  your a legend wanna share discord bot commendbot?

**Comment by Noksiowski** [comment]
4 everyone that needs 1 line fix:
Noxiu#0793

**Comment by ninjaibiss** [comment]
added

**Comment by playboost** [comment]
we wait for the correct solution, temporary solution is: change value 5000 for 10000 line  263 /helper/accounts.js

**Comment by ninjaibiss** [comment]
you mean line 274? @playboost

**Comment by Waller1337** [comment]
@ninjaibiss add me on discord WALLER1337#8543

**Comment by ninjaibiss** [comment]
done

**Comment by playboost** [comment]
> you mean line 274? @playboost

yes!!!! my file is different from the original. tks...

**Comment by ninjaibiss** [comment]
@playboost  im having a problem my fetcher cant find anyone

**Comment by trustyuy** [comment]
u are in server ?

**Comment by ninjaibiss** [comment]
oh now its working butttt the the temp fix is not working

**Comment by playboost** [comment]
> oh now its working butttt the the temp fix is not working

I was testing and working again, it was not my changes.

**Comment by lightbgs** [comment]
> @lightbgs do you have a discord commendbot?

y, send me ur nick#tag

**Comment by lightbgs** [comment]
> ![image](https://user-images.githubusercontent.com/65316785/90642569-3d713a80-e23b-11ea-890c-9b54b206fa5d.png)
> it works ı will made a tutorial

so what you did?

**Comment by BeepIsla** [comment]
Duplicate #593

**Comment by BeepIsla** [comment]
Thread is a duplicate since it came after #593. So move your conversations over there, regardless answering some stuff.

---

> and as stated by valve it was "forever fixed"

Valve never said that: https://twitter.com/basisspace/status/1295916342701199360

Specifically read his replies to others `These types of problems are difficult or impossible to "solve" forever.`

---

> no one cares? you very brainless

Then don't comment

---

> thats the only change valve made to csgo
> optional .CMsgGCCStrike15_v2_MatchmakingGC2ServerReserve reservation = 2;

This is irrelevant

---

> jaeger is selfish he dont tell fix

The fix is right there in that screenshot

---

> I have a discordcommendbot but still on beta maybe u can help me improve it?

This is entirely off-topic.

**Comment by medhatmo** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/31590513/90633460-750cc200-e1fc-11ea-9d1e-3f5e076d4f74.png)

can i have yuor discord plz

**Comment by medhatmo** [comment]
> 
> 
> ![image](https://user-images.githubusercontent.com/65316785/90642569-3d713a80-e23b-11ea-890c-9b54b206fa5d.png)
> it works ı will made a tutorial

brother plz can talk u discord medhat2020#6917  plz man can give me yur discord thank u
